pnpm-lock.yaml

```diff
@@ -18,8 +18,8 @@ importers:
         specifier: ^5.0.6
         version: 5.0.6
       '@frontify/fondue':
-        specifier: 12.0.0-beta.316
-        version: 12.0.0-beta.316(@types/node@18.17.12)(@udecode/plate@21.5.0)(framer-motion@10.12.12)(react-dom@18.2.0)(react@18.2.0)(tailwindcss@3.3.3)(ts-node@10.9.1)
+        specifier: 12.0.0-beta.319
+        version: 12.0.0-beta.319(@babel/core@7.22.9)(@types/node@18.17.12)(react-dom@18.2.0)(react@18.2.0)(slate-history@0.93.0)(slate-hyperscript@0.77.0)(styled-components@5.3.6)(tailwindcss@3.3.3)(ts-node@10.9.1)(zustand@3.7.2)
```

Comment:
Looks like it add multiple versions of Fondue, which will break the contexts (need to check it deeper)

Reply:
fixed by adding `babel/core` to `sidebar-settings`